server/aesd_ioctl.h

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
No need to duplicate this header file, you can use the same file for both driver and app.